lib/quick-secrets/secret/manager.rb

```diff
@@ -79,7 +79,12 @@ def store(secret, password)
         #key = Digest::SHA256.hexdigest encrypted
         secret_uuid = gen_uuid()
         if store_secrets?
-          qs_db[:secret].insert(uuid: secret_uuid, initialization_vector: iv, encrypted_data: encrypted, expiration_date: nil)
+            db_iv = iv.unpack('H*')[0]
```

Comment:
IMO, the code is becoming increasingly hard to read (no comments), not DRY, and more POD-like.  Why not break out the DB code to separate classes that handle insertion and retrieval from the DB, allowing this code to pass arbitrary binary data.  That way you aren't also making any assumptions about the DB in this code.